diff --git a/tabled/Cargo.toml b/tabled/Cargo.toml
index 02bb347..fd9ec4f 100644
--- a/tabled/Cargo.toml
+++ b/tabled/Cargo.toml
@@ -317,8 +317,8 @@ ansi = ["papergrid/ansi", "ansi-str", "ansitok", "std"]
 macros = ["std"]
 
 [dependencies]
-papergrid = { version = "0.13", default-features = false }
-tabled_derive = { version = "0.9", optional = true }
+papergrid = { path = "../papergrid", default-features = false }
+tabled_derive = { path = "../tabled_derive", optional = true }
 ansi-str = { version = "0.8", optional = true }
 ansitok = { version = "0.2", optional = true }
 
